cds_ils/migrator/api.py

```diff
@@ -114,8 +118,33 @@ def import_parents_from_file(dump_file, rectype, include):
                     record = import_record(parent, model, provider)
                     records.append(record)
                 elif rectype == "multipart" and has_volumes:
-                    record = import_record(parent, model, provider)
-                    records.append(record)
+                    existing_parent = None
+                    multipart_id = parent["_migration"].get("multipart_id")
+                    if multipart_id:
+                        if multipart_id in unindexed_series:
+                            existing_parent = unindexed_series[multipart_id]
+                        else:
+                            search = SeriesSearch().query(
+                                "bool",
+                                filter=[
+                                    Q("term",
+                                      _migration__serial_id=multipart_id),
+                                ],
+                            )
+                            results = search.execute()
+                            hits_total = results.hits.total.value
+                            if hits_total > 0:
+                                existing_parent = results.hits[0]
+                            else:
+                                unindexed_series[multipart_id] = parent
+                    if not existing_parent:
+                        record = import_record(parent, model, provider)
+                        if record:
+                            records.append(record)
+                    else:
+                        # TODO ensure the parent record is actually the same
+                        warn("Multipart parent record already exists,"
```

Comment:
I am worried about this `TODO` :) most of them, in software dev., they will stay there forever! Is there an issue?